variable/include/scipp/variable/comparison.h

```diff
@@ -12,8 +13,22 @@
 
 namespace scipp::variable {
 
-/// Tests if the unit, values (and variances where appropriate) of two
-/// Variables are within an absolute tolerance.
+/// Functions to provide numpy-like element-wise comparison
+/// between two Variables.
```

Comment:
This is the doxygen for `is_approx` below, new code has been inserted between doxygen and its function.